experimental/embeddings/scripts/evaluate_promises_sequence_tagging.py

```diff
@@ -0,0 +1,64 @@
+import pickle
+from pathlib import Path
+from typing import Optional
+
+import experimental.datasets.utils.misc
+import flair
+import torch
+import typer
+from experimental.defaults import RESULTS_PATH
+from experimental.embeddings.tasks import sequence_tagging as st
+from flair.embeddings import TransformerWordEmbeddings
+
+
+app = typer.Typer()
+
+
+def setup(device: str) -> None:
```

Comment:
do we reuse it for other models/tasks?

Reply:
It should be used in all scripts. And result score should correspond to our leaderboard as it appears.